Month2/week1/Month2-Project Week 1-Remembering the Bases/VideoGame/VideoGames.cpp

```diff
@@ -0,0 +1,75 @@
+#include "VideoGames.h"
+
+FVideoGame::FVideoGame()
+{
+	GameName = "";
+	StudioName = "";
+	DayCreation = 0;
+	MonthCreation = 0;
+	YearCreation = 0;
+}
+FVideoGame::FVideoGame(const std::string NewGameName, const std::string NewStudioName, const int NewDayCreation, const int NewMonthCreation, const int NewYearCreation) 
+{
+	GameName = NewGameName;
+	StudioName = NewStudioName;
+	DayCreation = NewDayCreation;
+	MonthCreation = NewMonthCreation;
+	YearCreation = NewYearCreation;
+}
+std::string FVideoGame::GetGameName()const
+{
+	return GameName;
+}
+std::string FVideoGame::GetStudioName()const
+{
+	return StudioName;
+}
+int FVideoGame::GetDayCreation()const
+{
+	return DayCreation;
+}
+int FVideoGame::GetMonthCreation()const
+{
+	return MonthCreation;
+}
+int FVideoGame::GetYearCreation()const
+{
+	return YearCreation;
+}
+std::string FVideoGame::SetGameName(const std::string NewGameName)
+{
+	GameName = NewGameName;
+}
+std::string FVideoGame::SetStudioName(const std::string NewStudioName)
+{
+	GameName = NewStudioName;
+}
+int FVideoGame::SetDayCreation(int NewDayCreation)
```

Comment:
Why are you returning an int on these getters?

1. They don't need to return anything.
2. They aren't returning anything anyway, this is dangerous code.